src/Aspire.Dashboard/Model/Otlp/LogFilter.cs

```diff
@@ -81,16 +84,31 @@ c switch
 
     private string? GetFieldValue(OtlpLogEntry x)
     {
-        return Field switch
+        if (Field == Loc[Dialogs.FilterFieldMessage])
```

Comment:
There is a bi-directional mapping between field and display name (`Field` here). We can keep the field names non-localized, but it will not affect the actual functionality of the component.